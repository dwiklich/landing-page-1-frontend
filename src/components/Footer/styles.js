import styled, { css } from 'styled-components';
import { Container as TextComponent } from '../TextComponent/styles';
import { Container as SectionComponent } from '../SectionContainer/styles';

export const Container = styled.footer`
  ${({ theme }) => css`
    text-align: center;
    border-top: 0.2rem solid ${theme.colors.mediumGray};
    a {
      color: inherit;
      text-decoration: none;
      font-size: ${theme.fonts.sizes.small};
    }

    & ${SectionComponent} {
      padding-top: 0;
      padding-bottom: 0;
    }

    & ${TextComponent} {
      font-size: ${theme.fonts.sizes.small};
    }
  `}
`;
