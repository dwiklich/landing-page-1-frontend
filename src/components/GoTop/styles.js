import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme }) => css`
    position: fixed;
    border-radius: 50%;
    display: flex;
    align-items: center;
    justify-content: center;
    background: ${theme.colors.primaryColor};
    color: ${theme.colors.white};
    width: 4rem;
    height: 4rem;
    bottom: 2rem;
    right: 2rem;
    z-index: 6;
  `}
`;
