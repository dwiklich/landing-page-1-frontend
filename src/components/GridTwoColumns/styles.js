import styled, { css } from 'styled-components';

export const Container = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: center;

    > img {
      width: 50rem;
    }
  `}
`;
