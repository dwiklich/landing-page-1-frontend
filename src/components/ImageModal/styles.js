import styled, { css } from 'styled-components';

/* The Modal (background) */
export const Container = styled.div`
  ${({ theme, modal }) => css`
    display: ${modal ? 'none' : 'block'}; /* Hidden by default */
    position: fixed; /* Stay in place */
    z-index: 1; /* Sit on top */
    padding-top: 100px; /* Location of the box */
    left: 0;
    top: 0;
    width: 100%; /* Full width */
    height: 100%; /* Full height */
    overflow: auto; /* Enable scroll if needed */
    background-color: rgb(0, 0, 0); /* Fallback color */
    background-color: rgba(0, 0, 0, 0.4); /* Black w/ opacity */
  `}
`;

/* Modal Content */
export const Content = styled.div`
  ${({ theme, modal }) => css`
    position: absolute;
    left: 50%;
    bottom: 50%;
    transform: translateX(-50%) translateY(50%);

    background-color: #fefefe;
    margin: auto;
    padding: 20px;
    border: 1px solid #888;
    width: 80%;

    @media ${theme.media.lteMedium} {
      width: 100%;
    }
  `}
`;

/* Modal Button */
export const ButtonClose = styled.span`
  ${({ theme, modal }) => css`
    color: #aaaaaa;
    float: right;
    font-size: 28px;
    font-weight: bold;

    &:hover,
    &:focus {
      color: #000;
      text-decoration: none;
      cursor: pointer;
    }
  `}
`;

export const Image = styled.img`
  ${({ theme }) => css`
    position: relative;
    left: 50%;
    transform: translateX(-50%);

    background-size: cover;
    width: 60%;
    border-radius: 1rem;
  `}
`;
