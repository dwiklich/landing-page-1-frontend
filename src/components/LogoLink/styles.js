import styled, { css } from 'styled-components';

export const Container = styled.a`
  ${({ theme, srcImage }) => css`
    display: flex;
    align-items: center;
    text-decoration: none;
    color: inherit;

    & > img {
      max-height: 3rem;
    }
  `}
`;
