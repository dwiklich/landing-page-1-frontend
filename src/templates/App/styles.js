import styled, { css } from 'styled-components';

export const Container = styled.div`
  /* ${(props) => css`
    background-color: ${props.color ? props.color : 'purple'};
  `} */

  /* ${({ theme, color }) => css`
    background-color: ${theme.colors.mainBg};
    color: ${color};
  `} */
`;
