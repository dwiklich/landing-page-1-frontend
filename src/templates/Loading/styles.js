import styled, { css, keyframes } from 'styled-components';

const rotate = () => keyframes`
  0%{
    transform: translate(-50%, -50%) rotate(0deg);
  }
  100% {
    transform: translate(-50%, -50%) rotate(360deg);
  }
`;

export const Container = styled.div`
  ${({ theme }) => css`
    position: absolute;
    top: 0;
    bottom: 0;
    left: 0;
    right: 0;
    float: 10;
    font-size: 3rem;
    background: ${theme.colors.primaryColor};
    color: ${theme.colors.white};

    &:before,
    &:after {
      content: '';
      position: absolute;
      top: 50%;
      left: 50%;
      transform: translate(-50%, -50%);
      border-width: 0.5rem;
      border-color: transparent;
      border-style: solid;
      border-radius: 50%;
    }

    &:after {
      width: 6rem;
      height: 6rem;
      border-left: 0.5rem solid ${theme.colors.secondaryColor};
      border-top: 0.5rem solid ${theme.colors.secondaryColor};
      animation: 1s ${rotate} infinite ease-in-out;
    }

    &:before {
      width: 2rem;
      height: 2rem;
      border-left: 0.5rem solid ${theme.colors.secondaryColor};
      border-top: 0.5rem solid ${theme.colors.secondaryColor};
      animation: 1s ${rotate} infinite ease-in-out reverse;
    }
  `}
`;
