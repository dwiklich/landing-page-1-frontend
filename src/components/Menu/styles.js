import styled, { css } from 'styled-components';
import { Container as SectionContainer } from '../SectionContainer/styles';
import { Title as Heading } from '../Heading/styles';

const menuVisible = (theme) => css`
  visibility: visible;
  opacity: 1;
`;

export const Container = styled.div`
  ${({ theme, visible }) => css`
    position: fixed;
    z-index: 5;
    top: 0;
    left: 0;
    right: 0;
    width: 100%;
    background: ${theme.colors.white};
    border-bottom: 1px solid ${theme.colors.mediumGray};
    transition: all 300ms ease-in-out;

    > ${SectionContainer} {
      padding-top: 0;
      padding-bottom: 0;
    }

    & ${Heading} {
      margin-top: 0;
      margin-bottom: 0;
    }

    @media ${theme.media.lteMedium} {
      /* display: ${visible ? 'flex' : 'none'}; */

      visibility: hidden;
      opacity: 0;

      ${visible && menuVisible(theme)}

      height: 100vh;

      > ${SectionContainer} {
        display: grid;
        grid-template-columns: 1fr;
        grid-template-rows: 1fr;
        height: 100vh;
        align-items: center;
        overflow-y: scroll;

        &::-webkit-scrollbar {
          display: none;
        }
      }

      & ${Heading} {
        padding-bottom: ${theme.spacings.large};
        display: flex;
        justify-content: center;
      }
    }
  `}
`;

export const MenuContainer = styled.div`
  ${({ theme }) => css`
    display: flex;
    justify-content: space-between;
    align-items: center;

    @media ${theme.media.lteMedium} {
      /* flex-direction: column; */
      display: block;
      text-align: center;
      padding: ${theme.spacings.xxlarge} 0;
    }
  `}
`;

export const Button = styled.button`
  ${({ theme, visible }) => css`
    z-index: 6;
    position: fixed;
    top: 2rem;
    right: 2rem;
    height: 4rem;
    width: 4rem;
    background: ${visible ? theme.colors.white : theme.colors.primaryColor};
    color: ${visible ? theme.colors.primaryColor : theme.colors.white};
    border: none;
    border-radius: 0.5rem;
    display: none;

    @media ${theme.media.lteMedium} {
      ${visible && menuVisible(theme)}

      display: flex;
      justify-content: center;
      align-items: center;

      > svg {
        width: 2.5rem;
        height: 2.5rem;
      }
    }
  `}
`;
