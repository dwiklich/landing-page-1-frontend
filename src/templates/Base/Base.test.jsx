import { screen } from '@testing-library/react';

import { renderTheme } from '../../styles/render-theme';
import { Base } from '.';

describe('<Base />', () => {
  it('should render', () => {
    renderTheme(<Base>texto</Base>);
    expect(screen.getByRole('heading', { name: 'texto' })).toBeInTheDocument();
  });
});
