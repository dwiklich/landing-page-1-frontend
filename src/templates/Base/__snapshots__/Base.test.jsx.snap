// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Base /> should render 1`] = `
.c12 {
  padding-top: 5.4rem;
}

.c4 {
  max-width: 120rem;
  margin: 0 auto;
  padding: 3.2rem;
  width: 100%;
}

.c7 {
  color: #0A1128;
  font-size: 2.4rem;
  text-transform: uppercase;
}

.c15 {
  color: #0A1128;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c20 {
  color: #0A1128;
  font-size: 3.2rem;
  text-transform: none;
}

.c22 {
  color: #FFFFFF;
  font-size: 6.4rem;
  text-transform: uppercase;
}

.c23 {
  color: #FFFFFF;
  font-size: 3.2rem;
  text-transform: none;
}

.c2 {
  position: fixed;
  z-index: 5;
  top: 0;
  left: 0;
  right: 0;
  width: 100%;
  background: #FFFFFF;
  border-bottom: 1px solid #DDDDDD;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c2 > .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c2 .c6 {
  margin-top: 0;
  margin-bottom: 0;
}

.c5 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-box-pack: justify;
  -webkit-justify-content: space-between;
  -ms-flex-pack: justify;
  justify-content: space-between;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c0 {
  z-index: 6;
  position: fixed;
  top: 2rem;
  right: 2rem;
  height: 4rem;
  width: 4rem;
  background: #0A1128;
  color: #FFFFFF;
  border: none;
  border-radius: 0.5rem;
  display: none;
}

.c8 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-text-decoration: none;
  text-decoration: none;
  color: inherit;
}

.c8 > img {
  max-height: 3rem;
}

.c10 {
  display: block;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
  padding: 1.6rem;
  color: #0A1128;
  position: relative;
}

.c10::after {
  content: '';
  position: absolute;
  bottom: 0.75rem;
  left: 50%;
  width: 0%;
  height: 0.2rem;
  background: #DC143C;
  -webkit-transition: all 300ms ease-in-out;
  transition: all 300ms ease-in-out;
}

.c10:hover::after {
  left: 25%;
  width: 50%;
}

.c9 {
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-flex-flow: row wrap;
  -ms-flex-flow: row wrap;
  flex-flow: row wrap;
}

.c1 {
  display: inline-block;
  vertical-align: middle;
  overflow: hidden;
}

.c17 {
  font-size: 2.4rem;
}

.c24 {
  text-align: center;
  border-top: 0.2rem solid #DDDDDD;
}

.c24 a {
  color: inherit;
  -webkit-text-decoration: none;
  text-decoration: none;
  font-size: 1.6rem;
}

.c24 .c3 {
  padding-top: 0;
  padding-bottom: 0;
}

.c24 .c16 {
  font-size: 1.6rem;
}

.c11 {
  position: fixed;
  border-radius: 50%;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
  -webkit-box-pack: center;
  -webkit-justify-content: center;
  -ms-flex-pack: center;
  justify-content: center;
  background: #0A1128;
  color: #FFFFFF;
  width: 4rem;
  height: 4rem;
  bottom: 2rem;
  right: 2rem;
  z-index: 6;
}

.c14 .c16 {
  margin-bottom: 6.4rem;
}

.c18 {
  counter-reset: grid-counter;
  display: grid;
  grid-template-columns: repeat(auto-fill,minmax(280px,1fr));
  gap: 3.2rem;
  overflow: hidden;
  width: 100%;
  padding: 2rem 1rem;
}

.c19 .c6 {
  position: relative;
  left: 5rem;
}

.c19 .c6:before {
  counter-increment: grid-counter;
  content: counter(grid-counter);
  position: absolute;
  font-size: 6.4rem;
  top: -3rem;
  left: -5rem;
  -webkit-transform: rotate(15deg);
  -ms-transform: rotate(15deg);
  transform: rotate(15deg);
}

.c13 {
  background: #FFFFFF;
  color: #0A1128;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

.c21 {
  background: #FFFFFF;
  color: #0A1128;
  background: #0A1128;
  color: #FFFFFF;
  min-height: 100vh;
  display: -webkit-box;
  display: -webkit-flex;
  display: -ms-flexbox;
  display: flex;
  -webkit-align-items: center;
  -webkit-box-align: center;
  -ms-flex-align: center;
  align-items: center;
}

@media (max-width:768px) {
  .c12 {
    padding-top: 0;
  }
}

@media (max-width:768px) {
  .c15 {
    font-size: 4rem;
  }
}

@media (max-width:768px) {
  .c22 {
    font-size: 4rem;
  }
}

@media (max-width:768px) {
  .c2 {
    visibility: hidden;
    opacity: 0;
    height: 100vh;
  }

  .c2 > .c3 {
    display: grid;
    grid-template-columns: 1fr;
    grid-template-rows: 1fr;
    height: 100vh;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
    overflow-y: scroll;
  }

  .c2 > .c3::-webkit-scrollbar {
    display: none;
  }

  .c2 .c6 {
    padding-bottom: 3.2rem;
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
  }
}

@media (max-width:768px) {
  .c5 {
    display: block;
    text-align: center;
    padding: 4.8rem 0;
  }
}

@media (max-width:768px) {
  .c0 {
    display: -webkit-box;
    display: -webkit-flex;
    display: -ms-flexbox;
    display: flex;
    -webkit-box-pack: center;
    -webkit-justify-content: center;
    -ms-flex-pack: center;
    justify-content: center;
    -webkit-align-items: center;
    -webkit-box-align: center;
    -ms-flex-align: center;
    align-items: center;
  }

  .c0 > svg {
    width: 2.5rem;
    height: 2.5rem;
  }
}

@media (max-width:768px) {
  .c9 {
    -webkit-flex-flow: column wrap;
    -ms-flex-flow: column wrap;
    flex-flow: column wrap;
    -webkit-align-content: center;
    -ms-flex-line-pack: center;
    align-content: center;
  }
}

@media (max-width:768px) {
  .c18 {
    grid-template-columns: 1fr;
  }
}

<div>
  <button
    aria-label="Open/Close menu"
    class="c0"
  >
    <svg
      aria-hidden="true"
      aria-label="Open menu"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M3 18h18v-2H3v2zm0-5h18v-2H3v2zm0-7v2h18V6H3z"
      />
    </svg>
  </button>
  <div
    aria-label="Menu Container"
    class="c2"
    role="dialog"
  >
    <div
      class="c3 c4"
    >
      <div
        class="c5"
      >
        <h1
          class="c6 c7"
        >
          <a
            class="c8"
            href="#"
          >
            Logo
          </a>
        </h1>
        <nav
          aria-label="Main menu"
          class="c9"
        >
          <a
            class="c10"
            href="#target1"
            target="_self"
          >
            Link 1
          </a>
          <a
            class="c10"
            href="#target2"
            target="_self"
          >
            Link 2
          </a>
          <a
            class="c10"
            href="#target3"
            target="_self"
          >
            Link 3
          </a>
          <a
            class="c10"
            href="#target4"
            target="_self"
          >
            Link 4
          </a>
          <a
            class="c10"
            href="#target5"
            target="_self"
          >
            Link 5
          </a>
          <a
            class="c10"
            href="#target6"
            target="_self"
          >
            Link 6
          </a>
          <a
            class="c10"
            href="#target7"
            target="_self"
          >
            Link 7
          </a>
        </nav>
      </div>
    </div>
  </div>
  <a
    aria-label="Go to top"
    class="c11"
    href="#"
    title="Go to top"
  >
    <svg
      aria-hidden="true"
      class="c1"
      fill="currentColor"
      focusable="false"
      viewBox="0 0 24 24"
      xmlns="http://www.w3.org/2000/svg"
    >
      <path
        d="M0 0h24v24H0V0z"
        fill="none"
      />
      <path
        d="M7.41 15.41L12 10.83l4.59 4.58L18 14l-6-6-6 6 1.41 1.41z"
      />
    </svg>
  </a>
  <div
    class="c12"
  >
    <div
      class="c13"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c14"
        >
          <h1
            class="c6 c15"
          >
            My grid
          </h1>
          <div
            class="c16 c17"
          >
            Lorem ipsum dolor sit,
          </div>
          <div
            class="c18"
          >
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c21"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c14"
        >
          <h1
            class="c6 c22"
          >
            My grid
          </h1>
          <div
            class="c16 c17"
          >
            Lorem ipsum dolor sit,
          </div>
          <div
            class="c18"
          >
            <div
              class="c19"
            >
              <h3
                class="c6 c23"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c23"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c23"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c13"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c14"
        >
          <h1
            class="c6 c15"
          >
            My grid
          </h1>
          <div
            class="c16 c17"
          >
            Lorem ipsum dolor sit,
          </div>
          <div
            class="c18"
          >
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c21"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c14"
        >
          <h1
            class="c6 c22"
          >
            My grid
          </h1>
          <div
            class="c16 c17"
          >
            Lorem ipsum dolor sit,
          </div>
          <div
            class="c18"
          >
            <div
              class="c19"
            >
              <h3
                class="c6 c23"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c23"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c23"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <div
      class="c13"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c14"
        >
          <h1
            class="c6 c15"
          >
            My grid
          </h1>
          <div
            class="c16 c17"
          >
            Lorem ipsum dolor sit,
          </div>
          <div
            class="c18"
          >
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 1
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 2
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
            <div
              class="c19"
            >
              <h3
                class="c6 c20"
              >
                Teste 3
              </h3>
              <div
                class="c16 c17"
              >
                Lorem ipsum dolor sit, amet consectetur adipisicing elit. Repellat iure porro in autem deserunt quis, voluptas hic est recusandae veniam modi, assumenda molestias et at eveniet vero eum error cumque?
              </div>
            </div>
          </div>
        </div>
      </div>
    </div>
    <footer
      class="c24"
    >
      <div
        class="c3 c4"
      >
        <div
          class="c16 c17"
        >
          <p>
            Teste de 
            <b>
              Footer
            </b>
          </p>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
